models/article-models.js

```diff
@@ -1,5 +1,15 @@
 const db = require("../db/connection");
 
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT article_id, title, topic, author, created_at, votes, article_img_url, CAST((SELECT COUNT(*) FROM comments WHERE articles.article_id=comments.article_id) AS INTEGER) AS comment_count FROM articles ORDER BY created_at`
```

Comment:
This nested query may not be necessary, look at the `sql-relationships` notes for COUNT